src/common/serverinfo.js

```diff
@@ -49,6 +49,12 @@ function getServerInfo(connection: Connection): Promise<GetServerInfoResponse> {
       reserveIncXrp: 'reserveIncrementXRP',
       seq: 'ledgerVersion'
     });
+    info.validatedLedger.baseFeeXRP =
+      info.validatedLedger.baseFeeXRP.toString();
```

Comment:
Why do tests pass? And I don't see conversion from drop to XRP. What I'm missing?